capstone-frontend/src/components/ListPage.jsx

```diff
@@ -34,34 +37,59 @@ class ListPage extends Component {
       gBooks.push(gBook[0]);
     }))
 
-    this.setState({ gBooks, loading: false });
+    this.setState({ gBooks, bookList: bookList[0], loading: false });
   }
 
-  componentDidMount() {
-    this.fetchBooks();
+  async componentDidMount() {
```

Comment:
What was the motivation for changing this? I'm not sure that this will change the behavior of the function.

Reply:
It just specifies to wait for the fetchBooks() to be done